app/views/partials/channels-tab-data-control.html

```diff
@@ -1,3 +1,7 @@
+<p>
+  Controls what data is stored and how requests and response are modified as they pass through the OpenHIM.
```

Comment:
_Controls what data is stored and how requests and response**s** are modified as they pass through the OpenHIM_